### DISCUSSION TITLE
Hardware and Master Internet Question

### DISCUSSION THREAD

**Discussion by Thormus** [Ysurac/openmptcprouter#3195] [Q&A] [answered]
Hi,
I'm giving OMR another go. 

DSL - 20/2 - very low ping
Verizon 5g - 300/20 - medium ping
Tmobile 5g - 300/70 - medium ping

Which internet connection should I use as the master?

What do you think about his hardware option? 
https://www.amazon.com/gp/product/B0CSK69VPB/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1

I appreciate any and all input. Also thank you to Ysurac for all your hardwork on this project.

**Comment by Ysurac** [comment, marked as answer]
DSL, because it's the connection that shouldn't lose too many packets.
I'm not a big fan of I22x-V, there are many strange problems with them.

**Reply by Thormus** [reply]
Ok thanks, any hardware that you would recommend that's 4 port 2.5ghz?

**Reply by Ysurac** [reply]
Anything not I22x-V (no problem when it's not a version V). Most problems I've seen in issues seems to be when DHCP is used.
I don't have any good models tested with ports 2.5G for now sorry, no idea.

**Reply by Thormus** [reply]
4gb or 8gb of ram?

https://www.amazon.com/gp/product/B07G9NHRGQ/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

**Reply by Ysurac** [reply]
4gb is enough. But it doesn't have 2.5G ethernet ports.

**Reply by Thormus** [reply]
Yeah I know but with my ISP's I likely won't be hitting anything near 1gb

**Reply by Thormus** [reply]
I wanted to try the n100 device but I can't get any of the x86-64 images to boot. Invalid sector size. Any recommendation on which image to use?

**Reply by Ysurac** [reply]
How do you copy the image on it ? and what image are you using ?

**Reply by Thormus** [reply]
Boot live Ubuntu with the commands listed in the install doc. Nvme0n1. I've tried the efi and non efi. Also tried squash and ext4. I tried the most recent beta but got kernel panics.

**Reply by Ysurac** [reply]
Did you try the 0.60rc1 kernel 6.1 ?

**Reply by Thormus** [reply]
I did. Both efi and non.

**Reply by Thormus** [reply]
I have omr running beautifully on the older device. I'm getting amazing stability and speed but my cpu is maxed out. That's why I'm trying the newer cpu.

**Reply by Ysurac** [reply]
What is the error/kernel panic you get ?

**Comment by Thormus** [comment]
V0.60beta1 works.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Ysurac** [reply]
And not v0.60rc1 ? really strange... I would really need the error.

**Comment by Thormus** [comment]
It flashes by so fast then kicks me out i don’t know how I’d capture it for you. Only words I can see as it happens is kernel panic

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Thormus** [comment]
rm /etc/hotplug.d/iface/00-nego fixes issues with 226-V. I'm fully operational and n100 is amazing. barely gets to 20% at full aggregation.

**Reply by Ysurac** [reply]
Strange, I'm not able to find this file...

**Reply by Thormus** [reply]
I'm on this version V0.60beta.
